code/modules/mob/living/carbon/human/life.dm

```diff
@@ -94,32 +94,49 @@
 	if(getBrainLoss() >= 60 && stat != DEAD)
 		speech_problem_flag = 1
 		if(prob(3))
-			var/list/s1 = list("IM A PONY NEEEEEEIIIIIIIIIGH",
+			var/list/s1 = list("IM A [pick("PONY","LIZARD","taJaran","kitty","Vulpakin","drASK","BIRDIE","voxxie","race car","combat meCH")] [pick("NEEEEEEIIIIIIIIIGH","sKREEEEEE","MEOW","NYA~","rawr","Barkbark","Hissssss","vROOOOOM","pewpew","choo Choo")]!",

```

Comment:
"IM A LIZARD choo Choo!"
"IM A combat meCH NYA~!"
"IM A drASK pewpew!"

These utterly random combinations really make this part of the feature realistic.

Reply:
Add spaceship (or spessship) to the first part please